dist/js/autocomplete.js

```diff
@@ -1065,7 +1066,11 @@
 		*/
 
 		setCache: function (key, data) {
-			return this.cache[(key || '').toLowerCase()] = data;
+			if (this.config.nocache) {
+				return;
```

Comment:
I'd at least return something here. Like maybe null. You can also remove the else statement since the if statement returns and exists the function.

if (this.config.nocache) {
    return null;
}
return this.cache[(key || '').toLowerCase()] = data;